app/fonts.styl

```diff
@@ -0,0 +1,64 @@
+@font-face {
```

Comment:
that's what I thought :) I'm not a big fan of leaving hacks to fix it in the future, often it's not enough time for that or nobody remember to change it - especially in big projects. But here `styles` are @natali-abyss 's responsibility, and if she changes it, it's ok for me :)